Library/ViewModels/ThanksViewModelTests.swift

```diff
@@ -80,11 +80,11 @@ final class ThanksViewModelTests: TestCase {
 
       goToDiscovery.assertValues([.illustration])
       XCTAssertEqual(
-        ["Thanks Page Viewed"],
+        ["Page Viewed"],
```

Comment:
This is fine but let's add another assertion to test the `context_type`. We should do this for all of the new event tickets since the event names have become more generic.